apache-rat-core/src/main/java/org/apache/rat/config/exclusion/ExclusionUtils.java

```diff
@@ -280,7 +280,7 @@ public static Optional<MatchPatterns> notMatchPattern(final Iterator<String> ite
 
     /**
      * @param filename file name to cleanup.
-     * @return
+     * @return cleaned up file name.
```

Comment:
@Claudenw the class has multiple unused methods - should we just remove them?
(same in other classes in this package/subpackage)